packages/components/src/toolbar/index.js

```diff
@@ -67,7 +67,7 @@ function Toolbar( { controls = [], children, className, isCollapsed, icon, label
 	}
 
 	return (
-		<ToolbarContainer className={ classnames( 'components-toolbar', className ) }>
+		<ToolbarContainer className={ className }>
```

Comment:
Hi @johngodley, in this file, we are in the components/toolbar component so I think it makes sense to add the class here like we do for the DropdownMenu component, as an alternative I think we should remove the class addition from the ToolbarContainer component.